client/my-sites/backup/backup-date-picker/index.tsx

```diff
@@ -23,8 +23,8 @@ const CALENDAR_DATE_CLICK = recordTracksEvent( 'calypso_jetpack_backup_date_cale
 
 const onSpace =
 	( fn: () => void ) =>
-	( { key }: { key?: string } ) => {
-		if ( key === ' ' ) {
+	( { code }: { code?: string } ) => {
```

Comment:
This is nice 👍 Let's just make sure that nothing is relying on the old behavior anymore.